extensions/http-curl/tests/VerifyInvokeHTTPTest.cpp

```diff
@@ -120,6 +126,31 @@ class VerifyInvokeHTTPOKResponse : public VerifyInvokeHTTP {
   }
 };
 
+class VerifyInvokeHTTPOK200Response : public VerifyInvokeHTTP {
+ public:
+  void runAssertions() override {
+    using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
+    assert(verifyLogLinePresenceInPollTime(std::chrono::seconds(6),
+        "key:invokehttp.status.code value:200",
+        "response code 200"));
```

Comment:
which handler responds with `200`? we redirect to `/`, does the civetweb have some default handler on `/`?

Reply:
It seems so. At first I wanted to make a separate handler for it, but after seeing it automatically returns `HTTP 200 OK` after redirecting to `/` and no other handler returns code `200` I decided to use that.